src/wrapper/wrapper.cpp

```diff
@@ -153,7 +156,11 @@ namespace
     options[METIS_OPTION_NUMBERING] = 0;  // C-style numbering
 
     if (contiguous)
-        options[METIS_OPTION_CONTIG] = 1;
+        options[METIS_OPTION_CONTIG] = contiguous;
+    if (ufactor > 0)
+        options[METIS_OPTION_UFACTOR] = ufactor;
+    if (seed > 0)
```

Comment:
Why these conditionals? Do we need to introduce a different "unset" state? Comment on why `<=0` cannot happen.

Reply:
Yeah, I wanted to leave a fallback to the previous behavior in case the values were not explicitly set. In this case there are no obvious defaults (I think METIS has some internal logic for setting them in case they are not explicity in the configuration). I guess that could also be done by passing ` bool <parameter>_set` values along with the set parameter values, or maybe there is a better way?

Reply:
IMO my previous suggestion (deal with the options array in Python) takes care of this, because we can tell more easily whether the argument was passed.